hw/ip/aes/data/aes.hjson

```diff
@@ -19,7 +32,7 @@
         starting encryption/decryption of the next block. Can only be updated
         when the AES unit is idle.
```

Comment:
Can you mention what happens if the engine is not idle?
"writes to the register while the engine is active are <dropped? cause error?>"

Reply:
The corresponding change is in PR #1053.